src/wallet/connection.ts

```diff
@@ -17,6 +17,7 @@ import {
 } from './types';
 
 export interface WalletServiceConnectionParams extends ConnectionParams {
+  network: string;
```

Comment:
I still don't understand what changed in your PR that you need to add this here, as far as I understood, `ConnectionParams` already has this parameter.

Reply:
Removed.